content/docs/dvclive/get-started.md

```diff
@@ -1,134 +1,129 @@
 # Get Started
 
 DVCLive is a Python library for logging machine learning metrics and other
-metadata in simple file formats, which is fully compatible with DVC.
+metadata in simple file formats which are fully compatible with DVC.
 
-<admon type="tip">
+## Set up DVCLive
 
-If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you
-can jump directly to its corresponding page.
+First of all, you need to add DVCLive to your Python code in order to start
+logging metrics.
 
-</admon>
-
-## Workflow
-
-### Initialize DVCLive
+In this example we are using [Keras](/doc/dvclive/ml-frameworks/keras) for
+training our model and we can add DVCLive as follows:
 
 ```python
-from dvclive import Live
-
-live = Live()
+# train.py
+from dvclive.keras import DvcLiveCallback
+...
+model.fit(
+  train_dataset, validation_data=validation_dataset,
+  callbacks=[DvcLiveCallback(path="training_metrics")])
 ```
 
-See [`Live()`](/doc/dvclive/api-reference/live) for details.
-
-### Log data
+You can find example snippets and reference of the options available in the
+corresponding page of any other of the supported
+[ML Frameworks](/doc/dvclive/ml-frameworks).
 
-<toggle>
-<tab title="Scalars">
+If you prefer to use the DVCLive API directly, check the
+[API Reference](/doc/dvclive/api-reference).
 
-```python
-live.log("acc", 0.9)
-```
+## Set up DVC
 
-See `Live.log()`.
+In order to visualize the outputs generated by DVCLive, we can set up a DVC
```

Comment:
Maybe it's better to say that you need DVC in order to track the outputs and compare experiments. You can already visualize the outputs without DVC.